src/components/DocumentationTopic.vue

```diff
@@ -502,6 +504,7 @@ export default {
     normalizePath(path) {
       // Sometimes `paths` data from `variants` are prefixed with a leading
       // slash and sometimes they aren't
+      if (!path) return path;
```

Comment:
Added this so we don't have to check if `path` is null everytime before we call this method.

Reply:
I can see why you did it this way. Personally, it feels a little odd to expect that `normalizePath` can be called without an actual path, although there's nothing technically wrong with it.